debian/control

```diff
@@ -589,7 +583,7 @@ Description: RADOS striping interface (development files)
 Package: librbd1
 Architecture: linux-any
 Section: libs
-Depends: librados3 (= ${binary:Version}),
```

Comment:
@dillaman any issues with CephContext or other structures here?

Reply:
Yeah -- they need to be version linked

Reply:
ahh, i see. we take advantage of the fact that `librados::config_t` is actually `CephContext*` in `(anonymous_namespace)::get_cct()`.